`show-names` feature  broken

<!--
Thanks for reporting a bug! ⛰

1. Make sure the bug is caused by Refined GitHub. Try disabling the extension first.
2. Include a full URL where the bug appears.
3. Include a screenshot/gif 

Issues without a URL/screenshot will be closed
-->
```
❌ show-names
content.js:1880 Error: GraphQL:
Parse error on "[" (LBRACKET) at [1, 61]
    at content.js:1992
```

https://github.com/highlightjs/highlight.js/pull/2023

@bfred-it it was fix but...

```
❌ show-names
content.js:1880 Error: GraphQL:
Could not resolve to a User with the login of 'greenkeeper'.
    at content.js:1992
```

It still errored out Since the link changed (its now marketplace)

--- a/source/features/show-names.tsx
+++ b/source/features/show-names.tsx
@@ -7,7 +7,7 @@ import {getUsername} from '../libs/utils';
 
 async function init(): Promise<false | void> {
 	// `a` selector needed to skip commits by non-GitHub users
-	const usernameElements = select.all('.js-discussion a.author:not(.rgh-fullname):not([href*="/apps/"])');
+	const usernameElements = select.all('.js-discussion a.author:not(.rgh-fullname):not([href*="/apps/"]):not([href*="/marketplace/"])');
 
 	const usernames = new Set();
 	const myUsername = getUsername();

